### DISCUSSION TITLE
CompositeDisposable now internal makes it more cumbersome to write custom Transitions

### DISCUSSION THREAD

**Discussion by llfab** [AvaloniaUI/Avalonia#10223] [General]
I had created a custom CrossFade that has improved handling for a ```TransitionalContentControl``` checking if ```Visual``` is a TCC with null content. I had basically copied an improved that from the orignial ```CroseFade``` that makes use of ```CompositeDisposable``` with that now being ```internal``` that is now more difficult. A solutions certainly is to also copy that one. 

Are you sure the trade-off having this internal good?

**Comment by jp2masa** [comment]
The class didn't exist before, Avalonia was using `System.Reactive.Disposables.CompositeDisposable`. Due to #9549 #9749, some classes from `System.Reactive` were copied as internal in order to remove the dependency on `System.Reactive`. So, to get the old behavior you can simply install the `System.Reactive` package.

**Reply by llfab** [reply]
Ahhh... Then I did obviously not look right. Thanks
